gpkitmodels/SP/SimPleAC/README.md

```diff
@@ -0,0 +1,225 @@
+
+\begin{equation}
+    C_{D_{induced}} = \frac{C_L^2}{\pi AR e}
+\label{e:cdinduced}
+\end{equation}
+
+```python
+        # Thrust and drag model
+        C_D_fuse = CDA0 / S
+        C_D_wpar = k * C_f * S_wetratio
+        C_D_ind  = C_L ** 2 / (np.pi * A * e)
+        constraints += [W_f >= TSFC * T_flight * D,
+                    D >= 0.5 * rho * S * C_D * V ** 2,
+                    C_D >= C_D_fuse + C_D_wpar + C_D_ind,
+                    V_f_fuse <= 10*units('m')*CDA0,
+                    Re <= (rho / mu) * V * (S / A) ** 0.5,
+                    C_f >= 0.074 / Re ** 0.2]
+```
+
+Fuel volume model
+-----------------
+
+The fuel volume model is the main difference between simpleWing and SimPleAC, and introduces the only signomial constraints in the model. Firstly we define the required fuel volume using fuel density $\rho_f$. 
+
+\begin{equation}
+    V_f = \frac{W_f } {\rho_f g}
+\label{e:vf}
+\end{equation}
+
+We consider wing fuel tanks and fuselage fuel tanks. The fuselage fuel was defined in the aerodynamic model, where the fuel volume contributes to drag. 
+
+\begin{equation}
+    V_{f_{wing}}^2 \leq 0.0009 \frac{S \tau^2}{AR}
+\label{e:vfwing}
+\end{equation}
+
+The signomial constraint is introduced here, where the available fuel volume is upper-bounded by the sum of the fuselage and wing fuel volumes. 
+
+\begin{equation}
+    V_{f_{avail}} \leq V_{f_{wing}} + V_{f_{fuse}}
+\label{vfavail}
+\end{equation}
+
+We constrain the total fuel volume to be less than the available fuel volume. 
+
+\begin{equation}
+    V_{f_{avail}} \geq V_{f}
+\label{e:vfineq}
+\end{equation}
+
+```python
+        # Fuel volume model 
+        with SignomialsEnabled():
+            constraints +=[V_f == W_f / g / rho_f,
+                V_f_wing**2 <= 0.0009*S**3/A*tau**2, 
+                    # linear with b and tau, quadratic with chord
+                V_f_avail <= V_f_wing + V_f_fuse, #[SP]
+                V_f_avail >= V_f]
+```
+
+Wing Weight Build-Up
+---------------
+
+The wing surface weight is a function of the planform area of the wing. 
+
+\begin{equation}
+W_{w_{surf}} \geq W_{w_{coeff2}} S
+\label{e:wwsurf}
+\end{equation}
+
+The wing structural weight is a complex posynomial expression that takes into account the root bending moment and shear relief due to presence of fuel in the wings. 
+
+\begin{equation}
+W_{w_{strc}}^2 \geq \frac{W_{w_{coeff1}}^2}{\tau^2} (N_{ult}^2 AR ^ 3 ((W_0+\rho_fgV_{f_{fuse}}) W S))
+\label{e:wwstrc}
+\end{equation}
+
+The total wing weight is lower-bounded. 
+
+\begin{equation}
+W_w \geq W_{w_{surf}} + W_{w_{strc}}
+\label{e:ww}
+\end{equation}
+
+```python
+        # Wing weight model
+        constraints += [W_w_surf >= W_W_coeff2 * S,
+            W_w_strc**2. >= W_W_coeff1**2./ tau**2. *
+            (N_ult**2. * A ** 3. * ((W_0+V_f_fuse*g*rho_f) * W * S)),
+            W_w >= W_w_surf + W_w_strc]
```

Comment:
the .md seems to miss returning the constraints? This markdown file should be able to fully replace the .py file, so that only one has to be maintained

Reply:
OK. That means that I will have to have all of the variable definitions in there too. Seems like a lot of clutter for doing a model write-up, but alright.

Reply:
You can skip them in the PDF by starting a python block with the comment `#inPDF: skip` (there's also `#inPDF: replace with $FOO.tex`, which is handy)